server/src/util.ts

```diff
@@ -15,7 +15,7 @@ let version: string | undefined
 /**
  * Try requiring
  */
-function tryRequire(path: string): any {
+export function tryRequire(path: string): any {
```

Comment:
There is another instance of this function in `client/src/test/helper.ts`, are they same? Can one of them be removed?

Reply:
They are the same, but since client and server should be separate and will be completely separate in future, I think I'll have to have both here.

Reply:
Sure, probably not important with just one function but if there are multiple of these util functions being duplicated, that can be extracted into its own package. Good for now 👍